R/correlation.R

```diff
@@ -247,6 +254,7 @@ correlation <- function(data,
                         select2 = NULL,
                         rename = NULL,
                         method = "pearson",
+                        use = c("pairwise.obs", "complete.obs"),
```

Comment:
```suggestion
                        missing = c("pairwise.obs", "complete.obs"),
```

Although I see the point of sticking with base-R argument names, `use` is a pretty vague one... Also, I'd move this argument to bethe last of the list as I think it's less "important"